test/exercism/user_test.rb

```diff
@@ -18,8 +18,13 @@ def test_flipper_id_is_username
     assert_equal 'spocktocat', user.flipper_id
   end
 
+  def build_github_user(id, username, email, avatar_url)
```

Comment:
I don't like having the verb in the name of the method. The method returns a GitHubUser; the method can be named `github_user`.